src/color.cpp

```diff
@@ -168,7 +169,7 @@ nc_color color_manager::get_invert( const nc_color col ) const
 nc_color color_manager::get_random() const
 {
     auto item = color_array.begin();
-    std::advance( item, rand() % num_colors );
+    std::advance( item, rng( 0, num_colors - 1 ) );
```

Comment:
Or just use `random_entry_ref` (there is an overload that takes a `std::array` like `color_array`). That way you don't have to deal with iterators or random index creation.